lib/web_ui/lib/src/engine/canvaskit/interval_tree.dart

```diff
@@ -0,0 +1,111 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.12
+part of engine;
+
+// TODO(hterkelsen): Tests.
+/// Associates a [T] with one or more [_UnicodeRange]s.
```

Comment:
This doc seems to describe the input used to construct the tree, but not the tree itself, which associates a number with a list of Ts.

Reply:
Done.